src/kOS/Execution/CPU.cs

```diff
@@ -238,7 +238,7 @@ public ProgramContext GetInterpreterContext()
             return contexts[0];
         }
         
-        public ProgramContext GetProgramContext()
+        public ProgramContext SwitchToProgramContext()
```

Comment:
I renamed this because it took me a while to find the problem, and part of the reason it took so long was that the original naming convention of "Get" ProgramContext hid the fact that actually there's a major side effect hiding inside this "get" method: Not only does it GET the program context, but it also tells the cpu to switch to starting to USE that program context as its next instruction it's going to execute after the current one is done.

That is crucial to how run() works.  I kept running around in circles trying to find "yes I see that run() loads the program, but where on earth does it tell the CPU to jump to start running it?  I can't find it anywhere!"

And the reason I couldn't find it is that it was hidden inside this "Get" method as a secret side-effect.  Thus I figured we really should rename it to make it clear it's not just doing a "Get".